prime-router/src/main/kotlin/cli/ValidateTranslationSchemaCommand.kt

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.fhirengine.translation.TranslationSchemaManager
+import org.apache.logging.log4j.kotlin.Logging
+
+class ValidateTranslationSchemaCommand :
+    CliktCommand(
+    name = "validateSchemas",
+    help = "Commands to validate translation schemas"
+),
+    Logging {
```

Comment:
This is just our linting rules but very confusing indentation!

```kotlin
class ValidateTranslationSchemaCommand : CliktCommand(
    name = "validateSchemas",
    help = "Commands to validate translation schemas"
), Logging {
```

Reply:
Yep, just how the linter formatted it